packages/malloy-db-bigquery/src/bigquery_connection.ts

```diff
@@ -769,6 +778,12 @@ export class BigQueryConnection
     return url;
   }
 
+  public getTableSourceUrl(tablePath: string) {
+    const tablePathInfo = tablePath.split('.');
+    const url = `https://console.cloud.google.com/bigquery?ws=!1m5!1m4!4m3!1s${tablePathInfo[0]}!2s${tablePathInfo[1]}!3s${tablePathInfo[2]}`;
+    return url;
+  }
+
```

Comment:
This implementation also sort of feels like it shouldn't be in our official API. Maybe the whole `BrowsableSourceConnection` stuff should just entirely be moved into the VSCode extension...

Reply:
removed